### PR TITLE
Explain events go inside target interfaces

### PR DESCRIPTION
**Pull request by a2sheppy**
Add information explaining that events are contained within the interfaces that receive them, and must be replicated in each such interface if there's more than one.

A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: docs/data-guidelines.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ For example, the `ImageData` constructor, `ImageData()`, is represented as `api.
 
 ## DOM events (`eventname_event`)
 
-Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
+Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`. Each event's record is a member of the interface which is its target. If an event can be sent to multiple interfaces, you need to replicate the event in each of those interfaces.
```
Comment (ddbeck) [inline docs/data-guidelines.md]: Can we present this an imperative? It's not optional, after all. Like this maybe?

```suggestion
Add DOM events as features of their target interfaces. If an event can be sent to multiple interfaces, then add the event as a feature of each interface. Name DOM event features in the form _eventname_\_event with description text `<code>eventname</code> event`.
```

File: docs/data-guidelines.md [hunk 2]
```diff
@@ -46,9 +46,9 @@ For example, the `ImageData` constructor, `ImageData()`, is represented as `api.
...
 
-For example, the feature for a `focus` event would be named `focus_event` with the description text `<code>focus</code> event`, like this:
+For example, the feature for a `focus` event whose target interface is `Element` would be named `focus_event` with the description text `<code>focus</code> event`, like this:
```
Comment (ddbeck) [inline docs/data-guidelines.md]: I'm not keen on anthropomorphizing events. How about this? 

```suggestion
For example, the feature for a `focus` event targeting the `Element` interface would be named `focus_event` with the description text `<code>focus</code> event`, like this:
```

Comment (a2sheppy) [reply docs/data-guidelines.md]: I've committed a revision that revises this text similarly to what you suggest.

File: docs/data-guidelines.md (hunk 1 above)
Comment (a2sheppy) [reply docs/data-guidelines.md]: I've rewritten the paragraph; hopefully it satisfies. It's not the same as what you wrote but is somewhat closer. :)

**Comment by a2sheppy** [comment]
That’s because I’m a professional. LOL :)

On July 17, 2019 at 7:25:07 AM, Daniel D. Beck (notifications@github.com)
wrote:

I like your rewrites better than mine! Thank you, Sheppy! 🎉


Eric Shepherd
Senior Technical Writer
MDN Web Docs <https://developer.mozilla.org/>
Blog: https://www.bitstampede.com/

**Comment by ddbeck** [comment]
I know I can't remember the last time I had a whole day of actual writing work. You don't have to rub it in 😆

**Pull request merged**